src/main/java/org/aion/wallet/ui/components/SendController.java

```diff
@@ -55,15 +63,27 @@ public void initialize(URL location, ResourceBundle resources) {
     @Subscribe
     private void handleAccountChanged(AccountDTO account) {
         this.account = account;
-        fromLabel.setText("From: " + account.getName());
+
+        accountAddress.setText(account.getPublicAddress());
+
+        accountBalance.setVisible(true);
+        accountBalance.setText(account.getBalance() + " " + AionConstants.CCY);
+        UIUtils.setWidth(accountBalance);
+
+        equivalentEUR.setVisible(true);
+        equivalentEUR.setText(Double.parseDouble(account.getBalance()) * AionConstants.AION_TO_EUR + " " + AionConstants.EUR_CCY);
```

Comment:
We could:
* Rename `BalanceFormatter` to `BalanceUtils`
* Add some conversion Method that converts from Aion to another CCY 
* Use that method here (and below) for conversion to different ccy

This way it would be easier when we get real values for exchange rates

Reply:
Created TODO, will be done in a future story